hydro_deploy/core/src/deployment.rs

```diff
@@ -186,3 +162,47 @@ impl Deployment {
         arc
     }
 }
+
+/// Buildstructor methods.
+#[buildstructor::buildstructor]
+impl Deployment {
+    #[allow(clippy::too_many_arguments)]
+    #[builder(entry = "GcpComputeEngineHost", exit = "add")]
```

Comment:
I wonder if it is possible to have `exit` be just `drop` (or is that asking for too much haha)?

Reply:
Hmm it is theoretically possible but the macro isn't implemented to support it -- also I think that would be _too_ fancy. Though the macro ought to mark the struct #[must_use] https://github.com/BrynCooke/buildstructor/issues/189

Reply:
Perhaps should be called `GcpComputeEngineHostBuilder` and `.build()`